app/src/main/AndroidManifest.xml

```diff
@@ -112,229 +109,214 @@
+                    android:scheme="https" />
                 <data
                     android:host="gist.githubusercontent.com"
-                    android:scheme="https"/>
-
-                <data android:pathPattern=".*"/>
+                    android:scheme="https" />
+                <data android:pathPattern=".*" />
 
-                <category android:name="android.intent.category.DEFAULT"/>
-                <category android:name="android.intent.category.BROWSABLE"/>
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
             </intent-filter>
             <intent-filter>
-                <action android:name="android.intent.action.SEND"/>
-                <data android:mimeType="text/plain"/>
-                <category android:name="android.intent.category.DEFAULT"/>
+                <action android:name="android.intent.action.SEND" />
+
+                <data android:mimeType="text/plain" />
+
+                <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
 
         <service
             android:name=".provider.tasks.notification.NotificationSchedulerJobTask"
             android:exported="false">
             <intent-filter>
-                <action android:name="com.firebase.jobdispatcher.ACTION_EXECUTE"/>
+                <action android:name="com.firebase.jobdispatcher.ACTION_EXECUTE" />
             </intent-filter>
         </service>
-
-        <service android:name=".provider.tasks.notification.ReadNotificationService"/>
-        <service android:name=".provider.tasks.git.GithubActionService"/>
-        <service android:name=".provider.tasks.git.ReactionService"/>
-        <service android:name=".provider.tasks.slack.SlackInvitationService"/>
-        <service android:name=".provider.tasks.version.CheckVersionService"/>
+        <service android:name=".provider.tasks.notification.ReadNotificationService" />
+        <service android:name=".provider.tasks.git.GithubActionService" />
+        <service android:name=".provider.tasks.git.ReactionService" />
+        <service android:name=".provider.tasks.slack.SlackInvitationService" />
+        <service android:name=".provider.tasks.version.CheckVersionService" />
         <service
             android:name=".provider.fcm.PushNotificationService"
             android:exported="false">
             <intent-filter>
-                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
             </intent-filter>
         </service>
 
         <meta-data
             android:name="io.fabric.ApiKey"
-            android:value="6ed82b6e0756853d7d782a3f547f84f9ecba217e"/>
+            android:value="6ed82b6e0756853d7d782a3f547f84f9ecba217e" />
 
+        <activity android:name=".ui.modules.search.SearchUserActivity"></activity>
```

Comment:
`/>` is enough and please if you having different code format, don't format the whole file instead just format the line you added.